### DISCUSSION TITLE
Project updates

### DISCUSSION THREAD

**Discussion by SimplyCorbett** [Joshua-Riek/ubuntu-rockchip#1079] [General]
I just want to confirm with Joshua taking a break that the project isn't dead / vaporware.

I basically just want to verify security updates will continue for the Orange Pi 5 plus. If not, thats going to suck. I don't particularly care if new features are added.

Thanks!

**Comment by Joshua-Riek** [comment]
I'm taking a break. I'll be releasing Ubuntu 24.10 later this month, the work is already done.
